From ac72ba274ea91e32bee4691d0ca0b1213a3ea95c Mon Sep 17 00:00:00 2001
From: Charlie Johnston <charlie.johnston@ni.com>
Date: Mon, 9 May 2022 11:03:55 -0500
Subject: [PATCH] Adding ja_JP.WINDOWS-31J to built locales

For Japanese locales, glibc only includes unicode-based locales right now but
some NI software requires non-unicode locales. This enables the WINDOWS-31J
(CP932) Japanese locale.

Upstream-Status: Inappropriate [NI-specific changes]
---
 localedata/SUPPORTED | 1 +
 1 file changed, 1 insertion(+)

diff --git a/localedata/SUPPORTED b/localedata/SUPPORTED
index 843ce78407..29b121122e 100644
--- a/localedata/SUPPORTED
+++ b/localedata/SUPPORTED
@@ -270,6 +270,7 @@ it_IT@euro/ISO-8859-15 \
 iu_CA/UTF-8 \
 ja_JP.EUC-JP/EUC-JP \
 ja_JP.UTF-8/UTF-8 \
+ja_JP.WINDOWS-31J/WINDOWS-31J \
 ka_GE.UTF-8/UTF-8 \
 ka_GE/GEORGIAN-PS \
 kk_KZ.UTF-8/UTF-8 \
-- 
2.34.1

